xilem_core/src/environment.rs

```diff
@@ -20,16 +20,20 @@ pub struct Rebuild;
 // --- MARK: Environment
 
 #[derive(Debug)]
-struct EnvironmentItem {
-    value: Box<dyn AnyDebug>,
+#[expect(missing_docs, reason = "Public on an interim basis")]
```

Comment:
Really on interim basis (same for the other changes here)?
I think since xilem (and other crates based on xilem_core) will likely have more views accessing the environment in specialized ways, that do not make sense(?) in xilem_core, that we can just leave this `pub`?

Or do you plan to further abstract this?

Reply:
I don't know! I don't expect this to be the final API for the environment.

That is, this PR is an exploration of the capabilities of the environment system, rather than the final state.